Lab6.ipynb

```diff
@@ -0,0 +1,692 @@
+      "text/plain": [
+       "<Figure size 1080x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#plotting the background distribution that I simulated with a true poisson pmf as a reference\n",
+    "x = np.arange(background_pmf.min(), background_pmf.max())\n",
+    "fig, ax = plt.subplots(1, 2, figsize=(15, 5))\n",
+    "ax[0].set_title('Simulated background for LHC data, mean=100')\n",
+    "ax[0].set_xlabel('Particle counts')\n",
+    "ax[0].set_ylabel('Number of occurrences')\n",
+    "ax[0].hist(background_pmf, bins=np.arange(0, 201), width=1, label='simulated background');\n",
+    "ax[0].plot(x, 100000*stats.poisson.pmf(x, 100), label='Poisson fit')\n",
+    "ax[0].legend()\n",
+    "ax[1].set_title('Simulated background of LHC data on a log scale, mean=100')\n",
+    "ax[1].set_xlabel('Particle counts (log)')\n",
+    "ax[1].set_ylabel('Number of occurrences')\n",
+    "ax[1].set_yscale('log')\n",
+    "ax[1].hist(background_pmf, bins=np.arange(0, 201), width=1, label='simulated background');\n",
+    "ax[1].plot(x, 100000*stats.poisson.pmf(x, 100), label='Poisson fit')\n",
+    "ax[1].legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "100,000 Poisson distribution simulation of the LHC background on a regular and a log scaled plot with a true Poisson pmf fitted to it. The fitted curve can be used for later analysis. Both the simulation and pmf have a mean of 100 particle counts."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to find the 5$\\sigma$ value of the simulated background, I must use the fitted pmf in order to be able to calculate the integral under the curve effectively. The integral corresponds to the probability of measuring a range of values, which is related to the 5$\\sigma$ threshold. Since I used the Poisson distribution (stats.poisson.rvs) to create 100,000 events with a mean count number of 100, the Poisson pmf (stats.poisson.pmf) with a mean of 100 fits the background. I find the number of counts on the Poisson pmf that, when integrated to $\\infty$,  gives a probability associated with measuring a signal 5$\\sigma$ away from the mean or greater of a Gaussian distribution with a mean of 0 and a standard deviation of 1. This gives the upper limit 5$\\sigma$, and to get the lower limit, I do the same thing, just with a probability of -5$\\sigma$ of lower. Since a Gaussian distribution is symmetric, these probabilities are the same. However, since a Poisson distribution is not symmetric except at very large N where the central limit theorem applies, the $\\pm 5\\sigma$ values might not be the same distance away from the mean."
```

Comment:
The distribution doesn't approach Guassian at large N data values generated. The distribution approaches a perfect Poisson with mean=100 at large N. The Poisson distribution approaches Guassian at a large mean, the can quickly be seen graphically if you plot the pmf with a mean of 1000.